internal/pkg/serialization/main.go

```diff
@@ -30,14 +30,34 @@ func SerializeCASCommand(key string, flags uint16, exptime int, size int, token
 	return casStream
 }
 
-// Creates the bytes stream for the delete operation
+// Creates the byte stream for the delete operation
 func SerializeDeleteCommand(key string) *bytes.Buffer {
 	del := fmt.Sprintf("delete %s\r\n", key)
 	delStream := bytes.NewBufferString(del)
 
 	return delStream
 }
 
```

Comment:
Looks good, both of the commands are checked for serialization, any others will throw an error